pkgs/development/python-modules/patchpy/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  more-itertools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage {
+  pname = "patchpy";
+  version = "2.0.4-unstable-2024-05-26";
```

```suggestion
  version = "2.0.3-unstable-2024-05-26";
```
https://github.com/MatthewScholefield/patchpy/blob/dcd673a0c2bf3f38e2b6b02e1a968be4f4d6c267/pyproject.toml#L3
or please pick `e00735b6a267bd8f3ed44b18a8c0cbab5c7bd0a0`